### PR TITLE
New currency format

### PR DESCRIPTION
**Pull request by idnovate**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.6.1.x
| Description?  | New currency format
| Type?         | new feature
| Category?     | CO
| BC breaks?    | N
| Deprecations? | N
| Fixed ticket? |
| How to test?  | Please indicate how to best verify that this PR is correct.

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
* Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/7823)

### CODE REVIEW

**Comment by maximebiloe** [comment]
Thanks for this contribution.
@AlexEven what do you think about this?

File: controllers/admin/AdminCurrenciesController.php [hunk 1]
```diff
@@ -151,7 +151,8 @@ public function renderForm()
                             array('key' => 2, 'name' => '0 000,00X ('.$this->l('Such as with Euros').')'),
                             array('key' => 3, 'name' => 'X0.000,00'),
                             array('key' => 4, 'name' => '0,000.00X'),
-                            array('key' => 5, 'name' => '0\'000.00X') // Added for the switzerland currency
+                            array('key' => 5, 'name' => '0\'000.00X'), // Added for the switzerland currency
+			    array('key' => 6, 'name' => '0.000,00X')
```
Comment (eternoendless) [inline controllers/admin/AdminCurrenciesController.php]: Please use four spaces instead of tabs for indentations.

**Review by eternoendless — CHANGES_REQUESTED** [review]
Thanks for this contribution! We just need a small change.

**Review by idnovate — COMMENTED** [review]
Tabs to spaces in controllers/admin/AdminCurrenciesController.php

**Comment by AlexEven** [comment]
@maximebiloe Can we add this in 1.6?
And is it that we need a new format, or should the "Euro" format be updated to include a dot as a thousand separator instead?
I don't know what's best / has the least impact for the version and users.

**Review by eternoendless — APPROVED** [review]
Thanks!

**Comment by idnovate** [comment]
:simple_smile::simple_smile:

**Comment by eternoendless** [comment]
@AlexEven I think it doesn't hurt to add it. Anyway, this will be better handled in 1.7.

**Comment by mickaelandrieu** [comment]
Except if I'm wrong we don't accept any new features on `1.6.1.x` branch which is in maintenance mode until the end of the year.

Ping @eternoendless

**Comment by eternoendless** [comment]
You are right @mickaelandrieu 

Unfortunately we cannot accept new features to the 1.6 branch. Sorry @idnovate!

**Pull request closed without merging**